server/src/main/java/org/opensearch/cluster/LocalNodeMasterListener.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
For new files, we don't need this part. This comment applies for all new files containing this licensing term.

Reply:
Thanks for your review! Because the file name doesn't change compared to before, so initially I didn't take them as new files. But it is reasonable to remove this part from license header.

Reply:
Removed those license header in commit https://github.com/opensearch-project/OpenSearch/pull/3914/commits/ebb2bc94a57cf8eedda72eb33bf8c985f8760d17. 😄

Reply:
@owaiskazi19 Ok, I will create a new PR to change the license header in main branch. Thanks for your advice!

Reply:
Created new PR to remove the part of license header in main branch https://github.com/opensearch-project/OpenSearch/pull/3924